src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="/styles/index.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="Photo of Apple iMac 27"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">
+        <span>Product code:</span>
+        <span data-id="id">195434</span>
+      </p>
+
+      <div class="card__rating">
+
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <span class="card__reviews">Reviews: 5</span>
```

Comment:
```suggestion
        <p class="card__reviews">Reviews: 5</p>
```

Reply:
Use semantic tags

And next time please check if there are some repeated mistakes in code, I've written about semantic tags during previous review, and you fix it only in one place. It could save time for both of us if you checked other text tags in your code 🙂

Reply:
Ok. 
Maybe it because I didn`t understand why do we need to use these tags on this type of element? I mean it is not the text block in our usual understanding. 
   But I will definitely take it into account! Thank you again)

Reply:
Done. And also done for other text-blocks;)

Reply:
It's a separated text block inside card, so it looks like paragraph 🙂